source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
I removed the references. See commit https://github.com/JeffSchering/docs/commit/8a0c801e5a236d9a7919310dbf793a6946feed52